paasta_tools/chronos_tools.py

```diff
@@ -478,10 +479,8 @@ def create_complete_config(service, job_name, soa_dir=DEFAULT_SOA_DIR):
     complete_config['description'] = get_config_hash(complete_config)
 
     # If the job was previously stopped, we should stop the new job as well
-    # NOTE this clobbers the 'disabled' param specified in the config file!
-    if desired_state == 'start':
-        complete_config['disabled'] = False
-    elif desired_state == 'stop':
+    # NOTE: this clobbers the 'disabled' param specified in the config file!
+    if desired_state == 'stop':
```

Comment:
If `get_disabled` looks at desired_state instead of being clobbered after the fact, then we can no longer differentiate a job a stopped job (paasta stop) with a 'disabled: True' key vs a stopped job without that key. This makes it impossible for emergency-stop to noop only for disabled jobs but not stopped ones.